### PR TITLE
Merge CB-2.32.0 branch into master

### PR DESCRIPTION
**Pull request by biharitomi**
See detailed description in the commit message.

### CODE REVIEW

**Review by brycederriso — APPROVED** [review]
DB backup/restore commits look good to me.

**Review by lajosrodek — CHANGES_REQUESTED** [review]

**Comment by lajosrodek** [comment]
DB SSL changes look good; only found one bug in `EntitlementServiceTest.java`.

File: auth-connector/src/test/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementServiceTest.java [hunk 1]
```diff
@@ -43,10 +43,8 @@
 
     static Object[][] entitlementCheckDataProvider() {
         return new Object[][]{
-                // entitlementName, function, enabled
-                {"CDP_AZURE", (EntitlementCheckFunction) EntitlementService::azureEnabled, false},
-                {"CDP_AZURE", (EntitlementCheckFunction) EntitlementService::azureEnabled, true},
```
[the remark this answers is not in the document]
Comment (biharitomi) [reply auth-connector/src/test/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementServiceTest.java:48]: Nice catch, thanks It was not a usual merge 😄

[the remark this answers is not in the document]
Comment (lajosrodek) [reply auth-connector/src/test/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementServiceTest.java:48]: Sure. Life would be too easy without (merge) conflicts. 😈

**Review by lajosrodek — APPROVED** [review]
LGTM

**Review by doktoric — APPROVED** [review]

**Pull request merged**